### PR TITLE
Dec21 stg release notes

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**

### CODE REVIEW

File: docs/whats-new/staging-release-notes.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
 
+## Staging release 21 December 2022
+Frontend version:  | Backend version: 
+
+**Changes** 
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Manage application role assignments now using TechBiz portal</summary>
+
+Tenants who have onboarded to TechBiz, need to use [TechBiz portal](https://portal.stg.techbiz.suite.gov.sg/) to manage the role assignments for it.
```
Comment (yqyeoh) [inline docs/whats-new/staging-release-notes.md:13]: to manage the role assignments for 'it' <-- 'it' here refers to? or shall we just say 'to manage application role assignments'

Also i checked with Lijia, they will only enforce this 4 Jan, so perhaps we add in front 'From 4 January 2022 onwards, ...'

Comment (SenthilKumarSivakami) [reply docs/whats-new/staging-release-notes.md]: done

File: docs/whats-new/staging-release-notes.md [hunk 2]
```diff
@@ -2,6 +2,19 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 21 December 2022
+Frontend version: 1.0.0-20221219.0756  | Backend version: 1.40.3-221219.0652
```
Comment (BananZG) [inline docs/whats-new/staging-release-notes.md:6]: sorry for last min change, the latest versions are: FE 1.0.0-20221221.0307 BE 1.40.4-221220.1019
```suggestion
Frontend version: 1.0.0-20221221.0307  | Backend version: 1.40.4-221220.1019
```

File: docs/whats-new/staging-release-notes.md [hunk 3]
```diff
@@ -2,6 +2,19 @@
...
+Frontend version: 1.0.0-20221221.0307  | Backend version: 1.40.4-221220.1019
+
+**Changes** 
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Manage application role assignments using TechBiz portal</summary>
+
+From 4 January 2023, if agencies can subscribe to an SGTS service (known as Tenant in TechPass portal) via the TechBiz portal, agencies need to use [TechBiz portal](https://portal.stg.techbiz.suite.gov.sg/) to manage their application role assignments.
+
+Currently, agencies manage application role assignments via TechPass portal or Automation API.
```
Comment (yqyeoh) [inline docs/whats-new/staging-release-notes.md:15]: currently agencies do not manage this in TP, it's the tenant themselves who manage this in TP. once the tenants are on TB, the agencies can manage it on their own on TB

Comment (SenthilKumarSivakami) [reply docs/whats-new/staging-release-notes.md]: done.

**Review by BananZG — APPROVED** [review]
LGTM

**Review by yqyeoh — APPROVED** [review]

**Pull request merged**